web/index.js

```diff
@@ -165,26 +164,33 @@ export async function createServer(
   }
 
   app.use("/*", async (req, res, next) => {
-    const shop = Shopify.Utils.sanitizeShop(req.query.shop);
-    if (!shop) {
+
+    if (typeof req.query.shop !== "string") {
```

Comment:
We still seem to be doing it after this block, but I wonder if there's a significant difference between `!shop` and `typeof req.query.shop !== "string"` in this case.

Can we ever get a non-string shop? If so, it might make more sense to guard against that in `sanitizeShop` than here.